services/QuillLMS/app/models/english_text.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+# == Schema Information
+#
+# Table name: english_texts
+#
+#  id         :bigint           not null, primary key
+#  text       :text             not null
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+class EnglishText < ApplicationRecord
+  has_many :translated_texts
+  has_many :translation_mappings
+
+  STANDARD_COMMENT = "
+  We are translating the instructions for an English-language grammar activity. The content of the activity itself is not translated. Therefore, please leave words that sound like they are part of the activity in the original english. Often they will between an HTML tag such as in <em>english word</em> or <ul>english  word</ul>.
+"
+
+  def self.translate!(jobs_list:)
+    resp = GengoAPI.postTranslationJobs(jobs: create_payload(jobs_list:))
+    return unless resp.present?
+
+    SaveTranslatedTextWorker
+    .perform_in(1.minute, resp.dig("response", "order_id"))
+  end
+
+  def self.save_translated_text!(order_id:)
+    response = GengoAPI.getTranslationJobs({order_id:})
+    response&.dig("response")&.each do |job|
+      create_translated_text!(job_id: job["job_id"])
+    end
+  end
+
+  def self.create_translated_text!(job_id:)
```

Comment:
This method does 3 things. Makes a request, parses the response, and creates the `translated_text`. Consider breaking this up into smaller methods that have one responsibility.